doc/classes/String.xml

```diff
@@ -670,8 +670,9 @@
 			</argument>
 			<description>
 				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				[b]Example:[/b] [code]"One,Two,Three,Four"[/code] will return [code]["Three","four"][/code] if split by [code]","[/code] with maxsplit of [code]"2"[/code].
+				If [code]maxsplit[/code] is specified, then it is number of splits to do from the right up to maxsplits, default is 0 which splits all the items.
```

Comment:
Since "maxsplit" does not take a "s" in the argument name, it should be kept as is. It should also always be referenced as `[code]maxsplit[/code]`.

Reply:
It would also split the sentence to be more readable, and improve grammar.
```
If [code]maxsplit[/code] is specified, it defines the number of splits to do from the right up to [code]maxsplit[/code]. The default value of 0 means that all items are split, thus giving the same result as [method split].
```